src/storage/mvcc/txn.rs

```diff
@@ -485,8 +485,34 @@ impl<S: Snapshot> MvccTxn<S> {
     }
 
     pub fn rollback(&mut self, key: Key) -> Result<bool> {
+        self.cleanup(key, 0)
+    }
+
+    /// Cleanup the lock if it's TTL has expired, comparing with `current_ts`. If `current_ts` is 0,
+    /// cleanup the lock anyway.
```

Comment:
`anyway` -> `at all events`?

Reply:
Sorry I didn't get it. How about `cleanup the lock without checking TTL`?

Reply:
Just feel `anyway` is weird here (I'm not a native English speaker, though)
`cleanup the lock without checking TTL` sounds good